docs/installation.md

```diff
@@ -0,0 +1,15 @@
+# Installation
+
+Now that you have downloaded the base system requirements, you will need to install the packages for the Gro API Client.
+
+Install with pip install
+
+```
+pip install git+https://github.com/gro-intelligence/api-client.git
+```
+Note: even if you are using [Anaconda](https://www.anaconda.com/), the API Client install should still be performed using pip and not [conda](https://docs.conda.io/en/latest/).
+
+Note: to find the location where Gro packages have been installed you can use the query:
+```
+pip show gro
+```

```

Comment:
Wouldn't hurt to also add a section at the bottom for updating to the latest version:

```
pip install --upgrade git+https://github.com/gro-intelligence/api-client.git
```

Reply:
done.